x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -50,7 +63,45 @@ export class Plugin
     this.alertTypeRegistry = alertTypeRegistry;
   }
 
-  public setup(): TriggersAndActionsUIPublicPluginSetup {
+  public setup(core: CoreSetup, plugins: PluginsSetup): TriggersAndActionsUIPublicPluginSetup {
+    const actionTypeRegistry = this.actionTypeRegistry;
+    const alertTypeRegistry = this.alertTypeRegistry;
+
+    this.managementApp = plugins.management.sections.section.insightsAndAlerting.registerApp({
```

Comment:
If I understand correctly, this adds the app to the section any way, and then the start contract then determines if it is enabled or disabled? If so, what's the initial state of an application: disabled or enabled?

Also, do we call each management screen an "application"? Maybe `managementPage` or something similar could be clearer, as we use the term "app" for top level kibana apps.

Reply:
> Also, do we call each management screen an "application"? Maybe managementPage or something similar could be clearer, as we use the term "app" for top level kibana apps.

We call these apps, too. I think "app" is the appropriate name to use because it's describing its nature (a primitive building block in our code and product, which is also unchanging), and not its location (which can change over time).

Reply:
>If so, what's the initial state of an application: disabled or enabled?

Enabled by default. Setting enabled / disabled is an optional step.

>Also, do we call each management screen an "application"?

We chose to call them apps in part because the API for them matches the top level API as much as possible.